### PR TITLE
chore: update pod deps' to match development team of 1st project

### PR DESCRIPTION
**Pull request by vharsh**
# Tl; dr:

This improves developer experience for iOS devs by reducing the clicks they need to do to make the project ready to run.

No further action required once the Development Team is selected for the `Inji.xcworkspace` project, i.e. no more of this 👇🏼 each time we do `pod install` as it'd work out of the box for everyone. 

Screenshot 2023-07-21 at 11 50 16 AM

# Detailed Summary

Before this change: Post running `npm install` & `pod install` the developers had to manually go and select the development team for three dependencies. This was a minor inconvenience, hence it was never automated.

After this change: Post running `npm install` & `pod install` the developer can just run `npm run ios` or push the play/archive button on XCode. The three Pod's config which used to require a valid `Development Team` will already have the `Development Team` set in the project's root without effort.

Local Testing: I've tested this locally on M1 Mac and this code has been taken from this StackOverflow answer so it's expected to be a good solution.

Any side effects: Since this meddles with build settings to build a project, this might be re-doing something already done in the build scripts in GH-Actions/Fastlane. This needs to be investigated appropriately.

Alternatives considered: This could have been done by hardcoding a Development Team, but this would have not been a super optimal approach IMO.

# TODO

- Test this to see if it breaks CI.

Credits: https://stackoverflow.com/a/75681139

### CODE REVIEW

**Review by dhivya0413 — APPROVED** [review]
LGTM

**Comment by vharsh** [comment]
We tried testing this on `ci-publish` and it seems that the builds got stuck for lack of MacOS runners(machine-type: `macos-13`)

**Comment by vharsh** [comment]
This was tested on `ci-publish` and it worked correctly.
